base/array.jl

```diff
@@ -213,7 +213,7 @@ function bitsunionsize(u::Union)
 end
 
 length(a::Array) = arraylen(a)
-elsize(::Type{<:Array{T}}) where {T} = aligned_sizeof(T)
+elsize(@nospecialize _::Type{A}) where {T,A<:Array{T}} = aligned_sizeof(T)
```

Comment:
`@nospecialize` is probably not allowed here?

Reply:
You mean because of where it is in bootstrap or syntax wise? It worked and fixed the inference time issue, so that wasn't a problem.